addons/strings/script_component.hpp

```diff
@@ -10,223 +10,3 @@
 #endif
 
 #include "\x\cba\addons\main\script_macros.hpp"
-
-#define UTF8_TABLE [\
-    ["%20"," "],\
-    ["%21","!"],\
-    ["%22",""""],\
-    ["%23","#"],\
-    ["%24","$"],\
-    ["%26","&"],\
-    ["%27","'"],\
-    ["%28","("],\
-    ["%29",")"],\
-    ["%2a","*"],\
-    ["%2b","+"],\
-    ["%2c",","],\
-    ["%2d","-"],\
-    ["%2e","."],\
-    ["%2f","/"],\
-    ["%30","0"],\
-    ["%31","1"],\
-    ["%32","2"],\
-    ["%33","3"],\
-    ["%34","4"],\
-    ["%35","5"],\
-    ["%36","6"],\
-    ["%37","7"],\
-    ["%38","8"],\
-    ["%39","9"],\
-    ["%3a",":"],\
-    ["%3b",";"],\
-    ["%3c","<"],\
-    ["%3d","="],\
-    ["%3e",">"],\
-    ["%3f","?"],\
-    ["%40","@"],\
-    ["%41","A"],\
-    ["%42","B"],\
-    ["%43","C"],\
-    ["%44","D"],\
-    ["%45","E"],\
-    ["%46","F"],\
-    ["%47","G"],\
-    ["%48","H"],\
-    ["%49","I"],\
-    ["%4a","J"],\
-    ["%4b","K"],\
-    ["%4c","L"],\
-    ["%4d","M"],\
-    ["%4e","N"],\
-    ["%4f","O"],\
-    ["%50","P"],\
-    ["%51","Q"],\
-    ["%52","R"],\
-    ["%53","S"],\
-    ["%54","T"],\
-    ["%55","U"],\
-    ["%56","V"],\
-    ["%57","W"],\
-    ["%58","X"],\
-    ["%59","Y"],\
-    ["%5a","Z"],\
-    ["%5b","["],\
-    ["%5c","\"],\
-    ["%5d","]"],\
-    ["%5e","^"],\
-    ["%5f","_"],\
-    ["%60","`"],\
-    ["%61","a"],\
-    ["%62","b"],\
-    ["%63","c"],\
-    ["%64","d"],\
-    ["%65","e"],\
```

Comment:
it didn't like one of these unicode? things
but it's fine to just move into the sqf

Reply:
Moving this to SQF doesn't help, it now throws:
```bash
error: Pest Error:    --> 161:16
    |
161 |     ["%c2%b2","²"],\
    |                ^---
    |
    = expected EOI, word, alpha, digit, underscore, dash, assignment, plus, left_brace, right_brace, left_bracket, right_bracket, left_parentheses, right_parentheses, colon, semicolon, join, directive, escape, slash, comma, decimal, double_quote, single_quote, left_angle, right_angle, unicode, newline, WHITESPACE, or COMMENT
```

Reply:
https://github.com/BrettMayson/HEMTT/pull/504
Should do the trick here

Reply:
Confirmed working. :+1:

Reply:
Fixed in HEMTT v1.5.2.